runtests

```diff
@@ -5,8 +5,8 @@ set -o pipefail
 
 main() {
   # abort if there are uncommitted local changes
-  msg="You have uncommitted changes. Not proceeding, as this script changes some files in place."
-  git diff-index --quiet HEAD -- || (echo "$msg" && exit 1)
+  #msg="You have uncommitted changes. Not proceeding, as this script changes some files in place."
+  #git diff-index --quiet HEAD -- || (echo "$msg" && exit 1)
```

Comment:
I wonder if we should add a check to bypass the changes check so that `./runtests` may be used more easily while working on a change 🤔